test/storage_project_test.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2020 The Bulward Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	controllerruntime "sigs.k8s.io/controller-runtime"
+
+	corev1alpha1 "github.com/kubermatic/bulward/pkg/apis/core/v1alpha1"
+	storagev1alpha1 "github.com/kubermatic/bulward/pkg/apis/storage/v1alpha1"
+	"github.com/kubermatic/utils/pkg/testutil"
+)
+
+func init() {
+	utilruntime.Must(corev1alpha1.AddToScheme(testScheme))
+	utilruntime.Must(clientgoscheme.AddToScheme(testScheme))
+	utilruntime.Must(storagev1alpha1.AddToScheme(testScheme))
+}
+
+func TestCoreProject(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Make this a parallel test with `t.Parallel()` It should be default behavior unless this is strictly avoided. (You can run `go test  -parallel=1 ./test/...` to disable any parallelis)